### DISCUSSION TITLE
[Ask] 2 Request inside 1 template

### DISCUSSION THREAD

**Discussion by daffainfo** [projectdiscovery/nuclei-templates#2293] [Q&A] [answered]
Is it possible to do 2 different requests inside one template? For example

The first request: edit a file to input a payload
```
POST /file/editfile.php HTTP/1.1
Host: 127.0.0.1:8000

target=php/daffainfo.php&content=<?php phpinfo(); ?>
```

The second request: check the edited file, for example daffainfo.php
```
GET/file/daffainfo.php HTTP/1.1
Host: 127.0.0.1:8000
```

Sorry for bad english

**Comment by geeknik** [comment, marked as answer]
Yes, please see misconfiguration/put-method-enabled.yaml as your reference. 👍🏻

**Reply by ehsandeep** [reply]
https://nuclei.projectdiscovery.io/templating-guide/protocols/http/#request-condition
